schemas/2014-04-01-preview/deploymentTemplate.json

```diff
@@ -120,6 +120,8 @@
                   { "$ref": "http://schema.management.azure.com/schemas/2016-04-01/Microsoft.Network.json#/resourceDefinitions/dnszones_SOA"},
                   { "$ref": "http://schema.management.azure.com/schemas/2016-04-01/Microsoft.Network.json#/resourceDefinitions/dnszones_SRV"},
                   { "$ref": "http://schema.management.azure.com/schemas/2016-04-01/Microsoft.Network.json#/resourceDefinitions/dnszones_TXT"},
+                  { "$ref": "http://schema.management.azure.com/schemas/2016-03-01/Microsoft.Scheduler.json#/resourceDefinitions/jobCollections"},
+                  { "$ref": "http://schema.management.azure.com/schemas/2016-03-01/Microsoft.Scheduler.json#/resourceDefinitions/jobCollections_jobs"},
```

Comment:
Doesn't only the top level resources get exposed here?  A job is not a registered resource.